h2o-algos/src/main/java/hex/schemas/DTV3.java

```diff
@@ -19,6 +19,7 @@ public static final class DTParametersV3 extends ModelParametersSchemaV3<DTModel
                 "categorical_encoding",
                 "response_column",
                 "seed",
+                "distribution",
```

Comment:
@wendycwong what is the idea behind the distribution here? I see that we optimize entropy in splits.. the attribute is not used in code.